src/kirby/components/card/card.component.html

```diff
@@ -1,4 +1,4 @@
-<div [ngClass]="colortype">
+<div [ngClass]="colortype" (click)="select.emit()">
```

Comment:
We don't do it this way, make a function for that:
```suggestion
<div [ngClass]="colortype" (click)="onCardSelect()">
```